R/nonmem.R

```diff
@@ -610,7 +569,80 @@ rex::register_shortcuts("babelmixr2")
     .extra <- paste0(paste(.prefixLines, collapse="\n"), "\n")
     rxode2::rxAssignControlValue(ui, ".nmPrefixLines", NULL)
   }
-  return(paste0(.extra, .rxToNonmemGetIndent(ui), .var, "=", .val))
+  paste0(
+    .extra,
+    .rxToNonmemGetIndent(ui), .var, "=", .val,
+    babelmixr2Deparse(x) # Show the source where this code came from
+  )
+}
+
+# When there is a more complex left-hand-side assignment (e.g. initial condition
+# setting or lag time setting)
+.rxToNonmemHandleAssignmentOperatorComplexLHS <- function(x, ui) {
+  stopifnot(length(x[[2]]) %in% c(2, 3))
+  if (length(x[[2]]) == 3) {
+    # Is d/dt() the only 3-long option?
```

Comment:
I think so...

Reply:
It'll error out if not, so we can find it then.